packages/vuetify/src/components/VDataIterator/VDataFooter.ts

```diff
@@ -141,7 +145,9 @@ export default Vue.extend({
 
         children = this.$scopedSlots['page-text']
           ? [this.$scopedSlots['page-text']!({ pageStart, pageStop, itemsLength })]
-          : [this.$vuetify.lang.t(this.pageText, pageStart, pageStop, itemsLength)]
+          : pageStart === pageStop + 1
```

Comment:
```suggestion
          : pageStart === pageStop
```

Reply:
I have the `+ 1` here because line 141 (`const pageStart = this.pagination.pageStart + 1`) increments `pagination.pageStart`, whereas lines 142-144 that set `pageStop` do not increment the value from `pagination`.

Reply:
Whatever is the reason for +1 it just didn't work for me and removing it fixed the problem.

### Playground
```vue
<template>
  <v-data-iterator
    :items="[{ id: '1', text: 'foo'}, {id: '2', text: 'bar'}]"
    item-key="id"
    :items-per-page="1"
    content-tag="v-layout"
    :footer-props="{ 'items-per-page-options': [1] }"
    #item="{ item }"
  >{{ item }}</v-data-iterator>
</template>
```

### Original
![image](https://user-images.githubusercontent.com/15625235/71050259-4b842e00-2177-11ea-9e3c-b551d3977e1e.png)

### Without +1
![image](https://user-images.githubusercontent.com/15625235/71050244-40c99900-2177-11ea-96d7-1067cf8a8ccd.png)

Reply:
I replaced this with logic that I think is both better and clearer.

Reply:
Is this expected behaviour?

![image](https://user-images.githubusercontent.com/15625235/71144912-392af280-2252-11ea-8b3b-12df46688af8.png)

```vue
<template>
  <v-data-iterator
    :items="[{ id: '1', text: 'foo'}, {id: '2', text: 'bar'}, {id: '3', text: 'baz'}]"
    item-key="id"
    :items-per-page="2"
    content-tag="v-layout"
    :footer-props="{ 'items-per-page-options': [2] }"
    #item="{ item }"
  >{{ item }}</v-data-iterator>
</template>
```